docs/en/install_deploy.md

```diff
@@ -0,0 +1,393 @@
+
+```bash
+$date
+Wed Aug 22 16:33:50 CST 2018
+```
+Please make sure the time is correct.
+
+## Deploy Standalone Version
+
+OpenMLDB standalone version needs to deploy a nameserver and a tablet. The nameserver is used for table management and metadata storage, and the tablet is used for data storage. APIServer is optional. If you want to interact with OpenMLDB in http, you need to deploy this module.
+
+**Notice:** It is best to deploy different components in different directories for easy upgrades individually.
+
+### Deploy tablet
+
+#### 1 Download the OpenMLDB deployment package
+
+```
+wget https://github.com/4paradigm/OpenMLDB/releases/download/v0.4.3/openmldb-0.4.3-linux.tar.gz
+tar -zxvf openmldb-0.4.3-linux.tar.gz
+mv openmldb-0.4.3-linux openmldb-tablet-0.4.3
+cd openmldb-tablet-0.4.3
+```
+
+#### 2 Modify the Configuration File: conf/standalone_tablet.flags
+
+* Modify `endpoint`. The endpoint is the deployment machine ip/domain name and port number separated by colons.
+
+```
+--endpoint=172.27.128.33:9527
+```
+
+**Notice:**
+* The endpoint cannot use 0.0.0.0 and 127.0.0.1.
+* If the domain name is used here, all the machines where the client using openmldb is located must be equipped with the corresponding host. Otherwise, it will not be accessible.
+
+#### 3 Start the service
+
+```
+sh bin/start.sh start standalone_tablet
+```
+
+**Notice**: After the service is started, the standalone_tablet.pid file will be generated in the bin directory, and the process number at startup will be saved in it. If the pid inside the file is running, the startup will fail.
+
+### Deploy Nameserver
+
+#### 1 Download the OpenMLDB deployment package
+
+```
+wget https://github.com/4paradigm/OpenMLDB/releases/download/v0.4.3/openmldb-0.4.3-linux.tar.gz
+tar -zxvf openmldb-0.4.3-linux.tar.gz
+mv openmldb-0.4.3-linux openmldb-ns-0.4.3
+cd openmldb-ns-0.4.3
+```
+
+#### 2 Modify the Configuration File: conf/standalone_nameserver.flags
+
+* Modify `endpoint`. The endpoint is the deployment machine ip/domain name and port number separated by colons.
+* The `tablet` configuration item needs to be configured with the address of the tablet that was started earlier.
+
+```
+--endpoint=172.27.128.33:6527
+--tablet=172.27.128.33:9527
+```
+
+**Notice**: The endpoint cannot use 0.0.0.0 and 127.0.0.1.
+
+#### 3 Start the service
+
+```
+sh bin/start.sh start standalone_nameserver
+```
+
+#### 4 Verify the running status of the service
+
+```bash
+$ ./bin/openmldb --host=172.27.128.33 --port=6527
+> show databases;
+ -------------
+  Databases
+ -------------
+0 row in set
+```
+
+### Deploy APIServer
+
+APIServer is responsible for receiving http requests, forwarding them to OpenMLDB and returning results. It is stateless and is not a must-deploy component of OpenMLDB.
+Before running, make sure that the OpenMLDB cluster has been started, otherwise APIServer will fail to initialize and exit the process.
```

Comment:
Before running =>
Before starting the APIServer

Reply:
Ok, thank you.